scripts/validate_deployment.py

```
@@ -0,0 +1,335 @@
+        """Test profiling enable/disable functionality."""
+        print("Testing profiling control...")
+        
+        # Test disable
+        result = self._make_request("POST", "network-profiling/disable")
+        if not result["success"] or result["status_code"] != 200:
+            print(f"❌ Disable profiling: FAIL - {result.get('error', 'Unknown error')}")
+            return False
+        
+        # Verify disabled
+        result = self._make_request("GET", "network-profiling/status")
+        if not result["success"] or not result["data"].get("enabled") == False:
```

Ruff (0.12.2)

Use `result["data"].get("enabled") != False` instead of `not result["data"].get("enabled") == False`

Replace with `!=` operator

(SIM201)